conans/test/functional/command/config_install_test.py

```diff
@@ -144,6 +146,19 @@ def _create_zip(self, zippath=None):
         zipdir(folder, zippath)
         return zippath
 
+    def _create_tgz(self, tgz_path=None):
```

Comment:
Maybe you can use the already existing ``tgz_with_contents()`` function in ``test/utils/test_files.py``?

Reply:
Thank you for the suggestion, done.